source/RoslynAnalyzers/Testing/Integration/IntegrationTestForwardCancellationTokenToInvocationsAnalyzer.cs

```diff
@@ -20,6 +20,9 @@ namespace Octopus.RoslynAnalyzers.Testing.Integration
     [DiagnosticAnalyzer(LanguageNames.CSharp)]
     public class IntegrationTestForwardCancellationTokenToInvocationsAnalyzer : DiagnosticAnalyzer
     {
+        // True / false, indicating whether a fix is suggested for the diagnostic.
+        internal const string OfferDiagnosticFix = "OfferDiagnosticFix";
```

Comment:
This is added to support edge case with static / local static function context. 
- Diagnostic is raised for this case, but no fix suggestion is provided. Considering this is an edge case, the value on supporting this now is minimal.